src/components/oauthDocs/acg/PKCEAuthContent.tsx

```diff
@@ -277,47 +245,24 @@ grant_type=authorization_code
         buttonSuccessMessage="Code updated!"
         theme="dark"
       />
-      <CodeWrapper>
-        <ReactMarkdown
-          rehypePlugins={[highlight]}
-          components={{
-            // eslint-disable-next-line react/display-name
-            code: ({ className, children, ...codeProps }): JSX.Element => (
-              // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-              <code tabIndex={0} className={className} {...codeProps}>
-                {children}
-              </code>
-            ),
-          }}
-        >
-          {`~~~json
+      <CodeBlock
+        withCopyButton
+        language="json"
+        code={`\
 {
     "access_token": "SlAV32hkKG",
     "expires_in": 3600,
     "refresh_token": "8xLOxBtZp8",
-    "scope": ${
-      props.apiDef?.oAuthInfo?.acgInfo?.scopes.join(' ') ?? 'profile openid offline_access'
-    },
+    "scope": ${scopes},
```

Comment:
Missing quotes around these scopes. Properly included further up the page.
```suggestion
    "scope": "${scopes}",
```

Reply:
Also this JSON should have 2 space indent